sites/main-page-title/_base.scss

```diff
@@ -9,6 +9,16 @@ h1#wb-cont, hgroup#wb-cont h1 {
 	border-top-width: 0;
 }
 
+h1#wb-cont[dir="rtl"], hgroup#wb-cont h1[dir="rtl"] {
```

Comment:
Missing an example with `hgroup`.

Should we expand the selector? like:
* `[dir="rtl"] h1#wb-cont`
* Not sure about this one, I suggest to wait for a use case before:  `hgroup[dir="rtl"]#wb-cont h1`

Reply:
example added. Thanks